recipe/meta.yaml

```diff
@@ -1,65 +1,58 @@
-{% set version = "2021.5.0" %}
+{% set version = "2021.6.0" %}
 {% set buildnumber = 0 %}
-{% set dal_version = "2021.5.1" %}    # [linux]
-{% set dal_version = "2021.5.0" %}    # [osx or win]
 
 package:
   name: daal4py
   version: {{ version }}
 
 source:
   url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
-  sha256: b0298ed384554077e8c49feb8092e895eb0f93dc98bc059cab8aede8a259acd8
+  sha256: a75acdc518f0af262a86952ee1040d5c8b9a9dbd4bf798fd974a917ee5b47e68
   patches:
     - patches/0001-replace-check_estimator_sparse_data-in-tests.patch
-    - winfix_ssize_t.patch  # [win]
-
+    # For patch 0002, see: https://github.com/python/cpython/issues/84006
+    - patches/0002-win-64-py37-distutils.patch  # [win and py==37]
 build:
   number: {{ buildnumber }}
-  # win-32 is not available
-  skip: True                  # [not ((win or linux or osx) and x86_64)]
-  skip: True                  # [py==310]
+  skip: True  # [not ((win or linux or osx) and x86_64)]
   missing_dso_whitelist:      # [osx]
     - lib/libc++.1.dylib      # [osx]
 
 requirements:
   build:
-    - {{ compiler('c') }}     # [not osx]
-    - {{ compiler('cxx') }}   # [not osx]
-    - cmake
-    - make   # [unix]
-    - ninja  # [win]
+    - make  # [linux or osx]
+    - {{ compiler('cxx') }}  # [not osx]
     - patch     # [unix]
     - m2-patch  # [win]
   host:
     - python
     - setuptools
-    - wheel
-    - pip
     - numpy
-    - dal-devel {{ dal_version }}
+    - dal-devel ==2021.6.0=hdb19cb5_916  # [linux]
+    - dal-devel ==2021.6.0=haf03e11_929  # [osx]
+    - dal-devel ==2021.6.0=h59b6b97_874  # [win]
     - cython
     - jinja2
-    - mpich                   # [not win]
+    - mpich      # [not win]
     - clang-tools
+    - cmake
     - pybind11
   run:
     - python
-    - dal {{ dal_version }}
+    - dal ==2021.6.0=hdb19cb5_916  # [linux]
+    - dal ==2021.6.0=haf03e11_929  # [osx]
+    - dal ==2021.6.0=h59b6b97_874  # [win]
     - {{ pin_compatible('numpy') }}
-  run_constrained:                                             # [osx]
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.12") }}  # [osx]
 
 test:
   requires:
```

Comment:
Should we add `test/imports`?
https://github.com/intel/scikit-learn-intelex/blob/650005cf20cb4cec994eda338fdd1fb47168f9c2/setup.py#L416-L435

Reply:
I'm inclined to leave those for now (following what the upstream maintainers test for).